From f5f8eaebb9140bc937c85061ef58d4750700ce0f Mon Sep 17 00:00:00 2001
From: Jeroen van der Heijden <jeroen@transceptor.technology>
Date: Tue, 28 Nov 2017 08:38:12 +0100
Subject: [PATCH] Add include

---
 include/siri/siri.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/siri/siri.h b/include/siri/siri.h
index ed4d77c..456e753 100644
--- a/include/siri/siri.h
+++ b/include/siri/siri.h
@@ -12,6 +12,7 @@
 #pragma once
 
 #include <uv.h>
+#include <pcre.h>
 #include <siri/grammar/grammar.h>
 #include <siri/db/db.h>
 #include <siri/file/handler.h>
-- 
2.15.0

